### ISSUE TITLE
Script runs but doesn't setup H/W transcoder on TrueNAS 12U1

### ISSUE THREAD

**Issue by PeteyHall** [danb35/freenas-iocage-plex#13]
When trying to execute the script under TrueNAS 12U1, I got an error that I would have to setup h/w transcoding manually even though I had set all of the variables up properly in my config file.  The script ran fine and the install still worked as expected, just h/w transcoding wasn't setup by default.

I have a Xeon e-2176g with the embedded igpu, which has been enabled in the BIOS.

**Comment by danb35** [comment]
@thorsteneb, any thoughts?

**Comment by yorickdowne** [comment]
Yeah I do. This is an upstream issue with lspci. Ix said this needed to be fixed upstream and they wouldn’t be pushing for it.

I can give you a PR that will a) work around the issue and b) slow that part of the script down by a few seconds because it’ll need to do a DNS lookup and fetch that information.

Do you want such a PR?

**Comment by yorickdowne** [comment]
#14 open for it
